test/e2e/cypress/fixtures/host-details/host_details.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Host details view
+ This is where the user has a detailed view of the status of one specific host in the deployed SAP System
+
+Background:
+    Given an healthy SAP deployment of 27 hosts having following agent ids and hostnames
+    # a09d9cf3-46c1-505c-8fb8-4b0a71a9114e => vmdrbdprd01
```

Comment:
Let's consider if we need this list or the simple detail of that one host we're testing against :wink: